package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-unimodules",
-  "version": "0.1.0-rc.2",
+  "version": "0.2.0-beta.0",
```

Comment:
Why `0.2.0`?

Reply:
Since now Android libraries will need to depend on `unimodules-core`, not `expo-core`